### PR TITLE
Portfolio Setup: HTML/CSS (Mobile First)

### PR DESCRIPTION
**Pull request by Kanyorok**
Made the mobile first HTML/CSS interface.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,47 @@
...
+    <main class="top_Section">
+      <section class="header">
+        <div class="app_bar">
+          <p>9:41</p>
+          <ul class="icons_phone">
+            <li><img src="icons/Cellular Connection.png" alt="Network Cell"></li>
+            <li><img src="icons/Wifi.png" alt="wifi"></li>
+            <li><img src="icons/Battery.png" alt="Battery"></li>
+          </ul>
+        </div>
```
Comment (braincee) [inline index.html]: - [ ] Please I highly recommend removing the div tag with `app_bar` class which is not part of the required figma design. 😃 

_**REQUIRED FIGMA DESIGN**_

Screenshot (238)


_**YOUR DESIGN**_

Screenshot (237)

File: index.html [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
+          <p>9:41</p>
+          <ul class="icons_phone">
+            <li><img src="icons/Cellular Connection.png" alt="Network Cell"></li>
+            <li><img src="icons/Wifi.png" alt="wifi"></li>
+            <li><img src="icons/Battery.png" alt="Battery"></li>
+          </ul>
+        </div>        
+        <nav class="menu_icon">
+          <ul>
+            <li>My Logo</li>
```
Comment (braincee) [inline index.html]: - [ ] Please I highly recommend wrapping the "My Logo" with the link  `a` tag for better semantics and user experience. 😄

File: index.html [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
+            <li><img src="icons/Union.png" height="12px" width="18px" alt="Menu"></li>
```
Comment (braincee) [inline index.html]: - [ ] Please I suggest wraaping the hamburger menu with `nav` tag for better HTML semantics to improve user experience with mobile devices. Please [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information. 😄

File: index.html [hunk 4]
```diff
@@ -1,15 +1,47 @@
...
+          <p class="desc">I can help you build a product, feature or website. 
+            Look through some of my work and experience! if you like
+            what you see and have a project you need coded, don't hesitate
+            to contact me. 
+          </p>
+          <ul>
+            <li><img src="icons/Github.png" alt="Github Account"></li>
+            <li><img src="icons/Linkedin.png" alt="Linkedin Account"></li>
+            <li><img src="icons/Twitter.png" alt="Github Account"></li>
+          </ul>
```
Comment (braincee) [inline index.html]: - [ ] Please I highly recommend adding link `a` tag on each `li` for HTML semantics correctly implemented. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Please [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information. 😄 
- [ ] I kindly suggest adding Medium and AngelList icons to the `li` to implement the figma design. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Kanyorok 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Kanyorok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**